pkgs/development/python-modules/yeelight/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchPypi, buildPythonPackage, future, enum-compat }:
+
+buildPythonPackage rec {
+  pname = "yeelight";
+  version = "0.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "8d49846f0cede1e312cbcd1d0e44c42073910bbcadb31b87ce2a7d24dea3af38";
+  };
+
+  propagatedBuildInputs = [ future enum-compat ];
```

Comment:
these probably are only needed for python2

Reply:
Probably, but the dependencies are declared in the install_requires of the setup.py. So the derivation throws an error if it can't import them

Reply:
that's fine then, although upstream should correctly package their own package